src/accelerator/ogl/util/device.cpp

```diff
@@ -100,6 +113,18 @@ struct device::impl : public std::enable_shared_from_this<impl>
 
         device_.setActive(false);
 
+#ifdef WIN32
+        if (d3d_device_) {
+            interop_handle_ = std::shared_ptr<void>(wglDXOpenDeviceNV(d3d_device_->device()), [](void* p) {
```

Comment:
This can segfault as wglDXOpenDeviceNV is not available on intel.
If this happens then just skip the whole shared texture stuff there, we can look into alternative api in a later PR.